python/dgl/transform.py

```diff
@@ -2720,45 +2720,63 @@ def as_immutable_graph(hg):
     return hg
 
 def sort_out_edges(g, tag, tag_offset_name='_TAG_OFFSET'):
-    """Return a new graph which sorts the out edges of each node.
+    r"""Return a new graph which sorts the out edges of each node.
```

Comment:
I feel that the docstring for `sort_out_edges` and `sort_in_edges` is a bit tricky because the changes are *under the hood* and not visible to users, as it merely reorders the `indices` array.  By saying `sorts the out edges` it somehow implies that the edge IDs or node IDs will be relabeled.  I don't have a better solution either though.